ambassador/tests/t_retrypolicy.py

```diff
@@ -0,0 +1,75 @@
+from kat.harness import Query
+
+from abstract_tests import AmbassadorTest, HTTP, MappingTest, ServiceType
+
+
+class RetryPolicyTest(MappingTest):
+    parent: AmbassadorTest
+    target: ServiceType
+
+    def init(self) -> None:
+        self.target = HTTP(name="target")
+
+    def manifests(self) -> str:
+        s = super().manifests() or ""
+
+        return s + """
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: retry
+spec:
+  selector:
+    app: retry
+  ports:
+  - port: 3000
+    name: http
+    targetPort: http
+  type: NodePort
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: retry
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: retry
+    spec:
+      containers:
+      - name: retry
+        image: live/ambassador-retry-service:1.0.0
```

Comment:
Do you mind committing the Dockerfile for this image with this PR as well? :)

Reply:
of course. initially i've added the code and dockerfile to the now unused end-to-end tests (but thats now excluded via gitignore). where should i put those files?